### ISSUE TITLE
Dashboard shows 431 Request Too Large error

### ISSUE THREAD

**Issue by drch-** [aws-samples/eks-workshop#1077]
Request headers > 8kb return an error.

Going through the Kubernetes Dashboard tutorial on eu-west-1 (Ireland)
In the Access the Dashboard tutorial, after logging in to the dashboard, the UI is briefly shown and then replaced with:

```
Unknown error (431)
Http failure response for https://xxx.vfs.cloud9.eu-west-1.amazonaws.com/api/v1/namespaces/kubernetes-dashboard/services/https:kubernetes-dashboard:/proxy/api/v1/token/refresh: 431 Request Header Fields Too Large 

```
Calls to /token/refresh seem to be failing because the header request size is > 8kb.  Total request header is 8.057 KB.  When this request fails, the UI is replaced with the error message.

Other calls are working fine and are just under the 8kb range.  I disabled the referer header in Firefox,  which brought the request header size down to 7.858 KB and everything worked fine.

**Comment by mzjung0** [comment]
@chunywan that workaround worked for me too thank you

**Comment by brippe** [comment]
I just started getting this 431 error on EKS (AWS) K8 1.21 and K8 Dashboard 2.4.0. I've uninstalled and reinstalled k8 dashboard with the instructions at https://www.eksworkshop.com/beginner/040_dashboard/dashboard/. I've cleared the browser's cache, tried different browsers and computers, but I still get a 431 error. When I try to login, I don't get past the login screen. If I refresh, I get a blank screen, thus, I can't login -> 431 error -> logout -> login. Any suggestions?
